### PR TITLE
add get for unsigned ints from univalue class

### PR DESCRIPTION
**Pull request by Greg-Griffith**
If I am requesting a merge into the wrong branch please let me know. I am not sure if this is the right place due to the lack of a clearly labeled dev branch.

### CODE REVIEW

**Comment by jgarzik** [comment]
* Yes, `master` is the correct dev branch.
* Travis failed, due to `nullptr` changes.
* Please either delete the `nullptr` changes, or move them to a separate and more-complete PR that turns on C++11 by default for this project.

**Comment by Greg-Griffith** [comment]
i reverted the nullptr changes back to NULL. I also added basic unit tests for this change in /test/object.

File: lib/univalue_get.cpp [hunk 1]
```diff
@@ -59,6 +59,22 @@ bool ParseInt64(const std::string& str, int64_t *out)
         n <= std::numeric_limits<int64_t>::max();
 }
 
+
+bool ParseUint64(const std::string& str, uint64_t *out)
```
Comment (promag) [inline lib/univalue_get.cpp]: Could be `static`.

File: lib/univalue_get.cpp [hunk 2]
```diff
@@ -59,6 +59,22 @@ bool ParseInt64(const std::string& str, int64_t *out)
...
+{
+    if (!ParsePrechecks(str))
+        return false;
+    char *endp = NULL;
+    errno = 0; // strtoll will not set errno if valid
```
Comment (promag) [inline lib/univalue_get.cpp]: Copy paste typos here and below? `strtoull ... `.

Comment (Greg-Griffith) [reply lib/univalue_get.cpp]: thanks, fixed

File: lib/univalue_get.cpp (hunk 1 above)
Comment (Greg-Griffith) [reply lib/univalue_get.cpp]: there is no difference either way. not worth changing

**Pull request closed without merging**